### PR TITLE
Last minute changes to various stuff

### PR DESCRIPTION
**Pull request by erasche**
- Update galaxy short intro for
  -  new CSS / defaults
  - new tool run description format
- Some last minute changes to qc/mapping
- updates to IGV launch help (thanks @shiltemann!)

### CODE REVIEW

File: topics/introduction/tutorials/galaxy-intro-101/tutorial.md [hunk 1]
```diff
@@ -160,17 +163,24 @@ Let's remind ourselves that our objective is to find which exon contains the mos
...
+>    {: .tip}
 >
->    > ### {% icon comment %} Comments
->    > **Note:** if you scroll down on this page, you will find the help of the tool.
->    {: .comment}
 >
-> 3. Click the **Execute** button and view the resulting file (with the {% icon galaxy-eye %} (eye) icon). If everything went okay, you should see a file that looks similar to this:
+> 3. Click **Execute**
+> 4. Wait for the job to finish
+> 5. View the resulting file (with the {% icon galaxy-eye %} (eye) icon). If everything went okay, you should see a file that looks similar to this:
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-101/tutorial.md]: I think these should now be numbered 2, 3, 4.

File: topics/introduction/tutorials/galaxy-intro-101/tutorial.md [hunk 2]
```diff
@@ -231,26 +243,32 @@ Now we have a list of all exons and the number of SNPs they contain, but we woul
...
->    ![Settings for the `Sort` tool](../../images/101_15.png)
+> 2. Click **Execute**
 >
-> 4. Click **Execute** and examine the output file.
+> 3. Examine the output file.
 >
 >    ![Contents of the `Sort` output dataset](../../images/101_16.png)
 >
->    You should now see the same file as we had before, but the exons with the highest number of SNPs are now on top.
+>    You should now see the same file as we had before, but the exons with the highest number of SNPs is now at the top.
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-101/tutorial.md]:
```suggestion
>    You should now see the same file as we had before, but the exons with the highest number of SNPs are now at the top.
```

File: topics/introduction/tutorials/galaxy-intro-101/tutorial.md [hunk 3]
```diff
@@ -373,40 +396,42 @@ We can examine the workflow in Galaxy's workflow editor. Here you can view/chang
...
 >
-> 4. **Click** on the box corresponding to the exon input dataset, and change the **Label** to `Exons` on the right-hand side of your screen.
+> 4. The box named `Exons` is named ok, but we want to change `SNPs` since this workflow is not specific to SNPs
 >
-> 5. **Repeat** this process for the other input dataset. Name it `Features`. We used it to calculate highest number of SNPs, but this workflow would also work with other features, so we give it a bit more generic name.
+>    - **Click** on the box corresponding to the `SNPs` input dataset
+>    - change the **Label** to `Features` on the right-hand side of your screen.
 >
-> 6. Let's also **rename the outputs**. Click on the `Select first` tool and in the menu on the right click on `Configure Output: 'out_file1'` and enter a descriptive name for the output dataset in the `Rename dataset` box.
+> 6. Let's also **rename the outputs**:
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-101/tutorial.md]:
```suggestion
> 5. Let's also **rename the outputs**:
```

Also the following steps need to be adjusted.

File: topics/introduction/tutorials/galaxy-intro-101/tutorial.md [hunk 4]
```diff
@@ -444,7 +471,10 @@ Now that we have built our workflow, let's use it on some different data. For ex
 >
 >    ![Settings for running the workflow](../../images/101_38.png)
 >
->    Once the workflow has started you will initially be able to see all its steps:
+>    > ### {% icon comment %} Potential workflow issues
+>    > Galaxy validates the workflow inputs to ensure they're correct. It may show a validation error at the start, until you select `Exons` for the Exons input, and your repeats for the Features input.
+>
+> 6. Once the workflow has started you will initially be able to see all its steps:
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-101/tutorial.md]:
```suggestion
> 6. Once the workflow has started, you will initially be able to see all its steps:
```

**Comment by bgruening** [comment]
Merci and good luck!

**Comment by erasche** [comment]
Danke

**Pull request merged**